modules/SofaGeneralLoader/MeshGmshLoader.cpp

```diff
@@ -253,7 +253,7 @@ bool MeshGmshLoader::readGmsh(std::ifstream &file, const unsigned int gmshFormat
                 nnodes = 10;
                 break;
             default:
-                serr << "Error: MeshGmshLoader: Elements of type 1, 2, 3, 4, 5, or 6 expected. Element of type " << etype << " found." << sendl;
+                msg_warning() << "Error: MeshGmshLoader: Elements of type 1, 2, 3, 4, 5, or 6 expected. Element of type " << etype << " found.";
```

Comment:
You can remove the Error:MesgGMeshLoader: prefix the is duplicated with msg_warning

Reply:
Fixed in 38a14dc